src/content/reference/kubectl-gs/get-nodepools.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "'kubectl gs get nodepools' command reference"
+description: Reference documentation on how to list node pools and get details for a single node pools using 'kubectl gs'.
+type: page
+weight: 10
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-ux
+---
+
+# `kubectl gs get nodepools`
+
+{{% kgs_alias_assumption %}}
+
+Like with all `get` commands in `kubectl`, this command can be used to get details on one item, a node pool in this case, or list several of them.
+
+## Usage
+
+### Get a list of node pools {#list}
+
+Simply execute
+
+```nohighlight
+kgs get nodepools
+```
+
+to list some information on all node pools available to you in the current installation.
+
+Here is some example output:
+
+```nohighlight
+ID      CLUSTER ID   CREATED                         CONDITION   NODES MIN/MAX   NODES DESIRED   NODES READY   DESCRIPTION
+ab12c   s921a        2021-01-02 15:04:32 +0000 UTC   READY       3/10            5               3             Production node pool
+```
+
+### Get specific node pool
+
+When used with a node pool ID as additional argument, the command will show details for a single node pool. Example:
+
+```nohighlight
+kgs get nodepools ab12c
+```
+
+Note: As an alternative to `get nodepools`, `get nodepool` will also work.
+
+## Output {#columns}
+
+The standard tabular output format features these columns:
+
+- `ID`: Unique identifier of the node pool.
+- `CLUSTER ID`: Unique identifier of the cluster that the node pool belongs to.
+- `CREATED`: Date and time of the node pool CR creation.
+- `CONDITION`: Latest condition reported for the node pool.
+- `NODES MIN/MAX`: Node pool autoscaler settings (if supported).
+- `NODES DESIRED`: The total number of nodes that the node pool should have.
+- `NODES READY`: The number of nodes in the node pool that are actually ready.
+- `DESCRIPTION`: User friendly description for the node pool.
+
+## Flags {#flags}
+
+Here we document the flags that have a particular meaning for the `get nodepools` command. Use `kgs get nodepools --help` for a full list.
+
+### `--cluster-id/-c` {#flags-output}
+
+If present, list the node pools that belong to this given workload cluster.
+
+### `--output/-o` {#flags-output}
+
+`kubectl` commonly allows to specify the output format for all `get` subcommands. `kgs get nodepools` is no different.
```

Comment:
> `kubectl` commonly allows to specify

I feel like we often use this grammatical sentence structure in our writing, and I think it's actually not right. I feel like the following would be more grammatically correct:

> kubectl allows _you_ to specify

or

> kubectl allows for specification of ...

I don't know if allow is even the best word in most cases where we use it. 
"Allow" implies permission and security or authorisation. 

In most cases where we use 'allow' we want to say that this is a capability or feature of the tool.

So I think "can" might be a better word.

> kubectl can specify the output format for...

Reply:
Also this sentence is really about how this command is no different from other `get` commands when it comes to output formatting. It's setting up a comparison between two things and saying that they are the same or similar. 

So perhaps we can  arrive quickly at the core idea of this sentence like:

"Similar to other `get` subcommands, you can specify the output format of `kgs get nodepools` using the `-o` flag"